lib/Experian/IDAuth.pm

```diff
@@ -299,13 +284,9 @@ sub _xml_as_hash {
 sub _get_result_proveid {
     my $self = shift;
 
-    my $report = $self->{result_as_xml} || die 'needs xml report';
+    my $report = $self->{result_as_xml} or die 'needs xml report';
```

Comment:
Somewhat inconsistent usage as there's `//` being used elsewhere.

Reply:
Generally in the code, the "maybe XML" strings are tested with `or`; a valid XML string will be a nonzero length and not "0", so this is already known boolean true.